fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm

```diff
@@ -92,13 +94,13 @@
 		return
 	if(owner.pulling == feed_target && owner.grab_state >= GRAB_AGGRESSIVE)
 		if(!IS_BLOODSUCKER(feed_target) && !IS_VASSAL(feed_target) && !IS_MONSTERHUNTER(feed_target))
-			feed_target.Unconscious((5 SECONDS + level_current))
+			feed_target.Unconscious((5 + level_current) SECONDS)
```

Comment:
I've recently tried to balance things around Bloodsuckers being equal in fighting at level 5 (so ~40 minutes into the game).

At this point;
- It's 10 seconds worth of sleep, at 25 units of blood (in an aggro grab)
- ``SSprocessing`` having a wait of 1 second, This 10 seconds will be 250 units of blood drained.

This leaves the person waking up with 310 cl of blood left (55% of your blood), which gives you about 10 lifeticks of breaking out of the grab, getting out of wherever the Bloodsucker has you trapped, and getting help, until you fall asleep of blood loss.

Do we want Feed to be this powerful?